src/prompts.js

```diff
@@ -142,15 +143,15 @@ const promptMainMenu = async (manifest) => {
       value: addEventListenerForEventProvider.bind(this, manifest, 'runtimeActions'),
     }
   )
-
-  if (!await isPluginExtensionInstalled()) {
-    choices.push(
-      {
-        name: "Subscribe to events automatically during deployment",
-        value: installPluginExtension.bind(this)
-      }
-    )
-  }
+// This functionality is desabled until the @adobe/plugin-extension ishttps://1-1-0-beta1-testing-kvixs6i-sgaxuvyy4jwem.us-4.magentosite.cloud/
```

Comment:
It's not a good practice to comment out code. It most probably won't be used ever again.
If it will be needed again we have a git history for it.

Reply:
Thank you! Will fix it now.